zipline/pipeline/factors/technical.py

```diff
@@ -598,7 +598,7 @@ class IchimokuKinkoHyo(CustomFactor):
         'kijun_sen_length': 26,
         'chikou_span_length': 26,
     }
-    inputs = USEquityPricing.high, USEquityPricing.close
+    inputs = [USEquityPricing.high, USEquityPricing.low, USEquityPricing.close]
```

Comment:
Just to be clear, as I was a little confusing: IchimokuKinkoHyo should have 3 default inputs but they added only 2, it was simply an oversight. I don't believe we need a test for this